### DISCUSSION TITLE
Upgrade 2.0.2 to 2.1.2 fails

### DISCUSSION THREAD

**Discussion by jabbrwcky** [kubernetes-sigs/cluster-api-provider-aws#4294] [Q&A]
Hi,

I just attempted to upgrade my management cluster capi-components using `clusterctl upgrade plan` and `AWS_B64ENCODED_CREDENTIALS=$(clusterawsadm bootstrap credentials encode-as-profile) clusterctl upgrade apply --contract v1beta1` but it fails to install the new aws-provider version claiming failure to reach my DNS server:

```
WARNING: `encode-as-profile` should only be used for bootstrapping.

Performing upgrade...
Scaling down Provider="infrastructure-aws" Version="v2.0.2" Namespace="capa-system"
Deleting Provider="infrastructure-aws" Version="v2.0.2" Namespace="capa-system"
Installing Provider="infrastructure-aws" Version="v2.1.2" TargetNamespace="capa-system"
Error: failed to connect to the management cluster: action failed after 9 attempts: Get "https://4C...7B.gr7.eu-central-1.eks.amazonaws.com/api?timeout=30s": dial tcp: lookup 4C..7B.gr7.eu-central-1.eks.amazonaws.com on 192.168.178.15:53: no such host
```

Is there a way to better find out what is happening or are there new requirements for 2.1.2?

**Comment by Skarlso** [comment]
Hmmm. There should be no new requirements in terms of the aws provider. That looks like something is not okay with a security group allowing access to the endpoint?
